minimint/src/lib.rs

```diff
@@ -209,24 +320,43 @@ impl MinimintServer {
         }
     }
 
+    fn start_next_epoch(&self, msg: &PeerMessage) -> bool {
```

Comment:
We might want to run a few epochs here, e.g. at least 1 every 10s for 100s (more if there are actual items) to allow the peer to join with some safety margin (they might want to join `next_epoch+5`).